test/extended/openstack/utils.go

```diff
@@ -46,6 +53,32 @@ type KuryrNetwork struct {
 	} `json:"status"`
 }
 
+type VolumeOption struct {
+	Name      string
+	PvcName   string
+	MountPath string
+}
+
+type DeploymentOpts struct {
+	Name     string
+	Labels   map[string]string
+	Replicas int32
+	Protocol v1.Protocol
+	Port     int32
+	Volumes  []VolumeOption
+}
+
+func NewDeploymentOpts() *DeploymentOpts {
```

Comment:
Every time you call this function, you then replace the values in the returned `DeploymentOpts`. Maybe you can remove this `NewDeploymentOpts` function and directly declare the `DeploymentOpts` from scratch every time you need it

Reply:
But I don't change all the values on every call